dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -0,0 +1,372 @@
+﻿#include "pch.h"
+#include "AppNotificationBuilder.h"
+#include <winrt/Windows.Globalization.h>
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationBuilder.g.cpp"
+#include <fmt/core.h>
+
+using namespace winrt::Windows::Globalization;
+using namespace winrt::Windows::Globalization::DateTimeFormatting;
+namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
+{
+    
+
+    std::wstring AppNotificationBuilder::GetWinSoundEventString(AppNotificationSoundEvent soundEvent)
+    {
+        static std::map<AppNotificationSoundEvent, std::wstring> c_soundEventMap
```

Comment:
If these are all string constants then `PCWSTR` would be better than `std::wstring` (no deep copy)

Reply:
Moving to switch statement